engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -437,7 +435,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
   }
 
   private def handleJobSuccessful(jobKey: JobKey, outputs: JobOutputs, data: WorkflowExecutionActorData) = {
-    workflowLogger.info(s"Job ${jobKey.tag} succeeded!")
+    workflowLogger.debug(s"Job ${jobKey.tag} succeeded!")
```

Comment:
Do people look for those pairings?

I don't, although I could see why a person would